packages/lodestar/src/opPool/opPool.ts

```diff
@@ -74,4 +76,39 @@ export class OpPool extends EventEmitter {
     ];
     await Promise.all(tasks);
   }
+
+  public async checkDuplicateProposer(config, block: BeaconBlock): Promise<void> {
+    const epoch: Epoch = computeEpochOfSlot(config, block.slot);
+    const proposers: Map<ValidatorIndex, Slot> = this.proposers.get(epoch);
```

Comment:
I think it should be block root instead of slot

Reply:
yeah tbh that makes way more sense to me